frontend/app/src/components/home/ChatMessage.svelte

```diff
@@ -109,11 +108,8 @@
     $: inert = deleted || collapsed;
     $: undeletingMessagesStore = client.undeletingMessagesStore;
     $: undeleting = $undeletingMessagesStore.has(msg.messageId);
-    $: canUndelete =
-        msg.content.kind === "deleted_content" &&
-        msg.content.deletedBy === user.userId &&
-        $now - Number(msg.content.timestamp) < 5 * 60 * 1000 && // Only allow undeleting for 5 minutes
-        !undeleting;
+
+    let canUndelete = true;
```

Comment:
Since this will almost always be false, shouldn't it default to false?
Otherwise it'll render as true, then re-render when it changes, I think?

Reply:
Oh wait doesn't this need to be a reactive variable?
So that when it changes those changes actually take effect.

Reply:
No it's fine as is since we are explicitly updating the value. It only needs to be reactive if it is computed from other values.